src/server/scripts/EasternKingdoms/SunkenTemple/instance_sunken_temple.cpp

```diff
@@ -35,7 +35,6 @@ enum Gameobject
     GO_ATALAI_STATUE4           = 148833,
     GO_ATALAI_STATUE5           = 148834,
     GO_ATALAI_STATUE6           = 148835,
-    GO_ATALAI_IDOL              = 148836,
     GO_ATALAI_LIGHT1            = 148883,
     GO_ATALAI_LIGHT2            = 148937
 
```

Comment:
Line 40 in your .cpp file (line 41 in the old version) is an unneeded blank line. Please remove it.